tests/validators/test_frozenset.py

```diff
@@ -157,12 +157,12 @@ def generate_repeats():
         (
             {'max_length': 3},
             infinite_generator(),
-            Err('Frozenset should have at most 30 items after validation, not 31 [type=too_long,'),
+            Err('Frozenset should have at most 3 items after validation, not >= 4 [type=too_long,'),
```

Comment:
This is because it used to hit the default 10x max_length "input" size, now we're measuring output size.